test/e2e/build_test.go

```diff
@@ -668,4 +668,20 @@ RUN ls /dev/test1`, ALPINE)
 		session.WaitWithDefaultTimeout()
 		Expect(session).Should(Exit(0))
 	})
+
+	It("podman build remote image test", func() {
```

Comment:
Is this test doing what you think it does? I ran it against current podman, and it passes. One requirement of a new test is that it fail in the previous version, because of course you're testing new functionality or a bug fix. This test does not seem to do that.

Reply:
I removed this test. The other e2e test covers the change fully.